src/IsNull.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
```

Comment:
Marijn usually wants these copyright notices all over the place, could you add them?